src/common/time/test/TimeParserTest.cpp

```diff
@@ -0,0 +1,249 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+#include <gtest/gtest.h>
+
+#include "common/base/Base.h"
+#include "common/time/TimeParser.h"
+
+TEST(TimeParser, DateTime) {
+  {
+    nebula::time::TimeParser parser;
+    auto result = parser.parseDateTime("2019-01-03T22:22:3.2333");
+    ASSERT_TRUE(result.ok()) << result.status();
+    EXPECT_EQ(nebula::DateTime(2019, 1, 3, 22, 22, 3, 233300), result.value());
+  }
+  // with offset
+  {
+    nebula::time::TimeParser parser;
+    auto result = parser.parseDateTime("2019-01-03T22:22:3.2333+02:30");
+    ASSERT_TRUE(result.ok()) << result.status();
+    EXPECT_EQ(nebula::DateTime(2019, 1, 3, 19, 52, 3, 233300), result.value());
+  }
+  // lack day
```

Comment:
Add case for `    auto result = parser.parseDateTime("2019-01-03T22:22:3.2333Z");` where `Z` stands for UTC

Reply:
Add in later